core/example/benchmark/Cabana_peakflops.cpp

```diff
@@ -320,11 +320,11 @@ void run()
 //---------------------------------------------------------------------------//
 int main( int argc, char **argv )
 {
-    Cabana::initialize(argc,argv);
+    Kokkos::initialize(argc,argv);
```

Comment:
We should consider teaching to use `Kokkos::ScopeGuard` instead.  It doesn't really matter here but it is a safer alternative because `Kokkos::finalize()` always gets called.  For instance if someone change the code and add an if-clause with early return (e.g. parsing user command line arguments and finding the flag `--help`)

Reply:
Good call - maybe do that in the tutorials?

Reply:
Yeah, I'll add that in the next commit